CHANGES.rst

```diff
@@ -19,7 +19,12 @@ Release History
    - Removed
    - Fixed
 
-0.1.0 (unreleased)
+0.1.0 (12/06/2016)
 ==================
 
-Initial release
+Initial release of Lasagne-based NengoDL
+
+0.2.0 (unreleased)
```

Comment:
We also do reverse chronological order for releases elsewhere (so newest to oldest). I think I just copied that organization from other projects, but people tend to want to know what's new in this release mostly, then ignore the rest, so there's some reasoning behind it.